CRM/Core/BAO/SchemaHandler.php

```diff
@@ -224,7 +225,7 @@ public static function buildIndexSQL(&$params, $separator) {
    *
    * @return bool
    */
-  public static function changeFKConstraint($tableName, $fkTableName) {
+  public static function changeFKConstraint($tableName, $fkTableName): bool {
```

Comment:
What is the benefit of returning `bool` if the only value the function can return is `TRUE`?

Reply:
:grin: Does a question cease being rhetorical if it is answered?

Reply:
Can any question truly be answered? 💭